README.md

```diff
@@ -15,7 +15,7 @@ pip install postgoose
 ## Usage
 
 ```
-usage: goose [-h] [--host HOST] [-p PORT] [-U USERNAME] [-d DBNAME] [-s SCHEMA] [-r ROLE] [-v] migrations_directory
+usage: goose [-h] [--host HOST] [-p PORT] [-U USERNAME] [-d DBNAME] [-s SCHEMA] [-r ROLE] [-m MIGRATIONTABLE]  [-v] migrations_directory
```

Comment:
How about using plural? `MIGRATIONS_TABLE` instead of `MIGRATIONTABLE`?

Reply:
acknowledged, I will make changes.